DEFAULT_CONFIG.json5

```diff
@@ -25,6 +25,24 @@
     endpoints: [
       // "<proto>/<address>"
     ],
+    //  /// Global connect retry configuration, 
+    //  /// The retry configuration can also be specified for the separate endpoint ("retry_" + field name)
+    //  /// it will override the global one
+    //  /// E.g. tcp/192.168.0.1:7447#retry_try_timeout_ms=20000;retry_period_max_ms=10000"
+    //  retry: { 
+    //    /// global timeout for full connect cycle (currently not supported)
```

Comment:
If it is not supported, let's remove it. We can always add it later.

Reply:
Did you mean remove the comment? Because this structure shared between connect and listen, and make no sense to split them.